packages/clay-color-picker/src/Hue.tsx

```diff
@@ -20,66 +20,56 @@ type Props = {
 	value: number;
 };
 
-const useIsomorphicLayoutEffect =
-	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
-
 /**
  * Renders Hue component
  */
 const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 	const containerRef = React.useRef<HTMLDivElement>(null);
-	const selectorActive = React.useRef<boolean>(false);
 
-	const {onPointerMove, setXY, x, y} = usePointerPosition(containerRef);
+	const {setXY, y} = usePointerPosition(containerRef);
```

Comment:
We don't need this hook anymore and we don't need to set `setXY` anymore because we are no longer using them here. It used to be just used to get `x` information to deal with the position of the pointer, as we don't have it anymore, we don't need it anymore.